### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'code_snippet')

### ISSUE THREAD

**Issue by FlyingBaum** [spatie/ignition#117]
**Please include some context and the contents of the console in your browser's developer tools.**

```
TypeError: Cannot read properties of undefined (reading 'code_snippet')
    at XE (http://localhost/:72:249981)
    at mi (http://localhost/:72:70706)
    at Ql (http://localhost/:72:122308)
    at xc (http://localhost/:72:109797)
    at _c (http://localhost/:72:109725)
    at Cc (http://localhost/:72:109588)
    at Nc (http://localhost/:72:106582)
    at hc (http://localhost/:72:103958)
    at nu (http://localhost/:72:119816)
    at http://localhost/:72:121187

Failed to load resource: the server responded with a status of 500 (Internal Server Error)

TypeError: Cannot read properties of undefined (reading 'code_snippet')
    at XE ((index):72:249981)
    at mi ((index):72:70706)
    at Ql ((index):72:122308)
    at xc ((index):72:109797)
    at _c ((index):72:109725)
    at Cc ((index):72:109588)
    at Nc ((index):72:106582)
    at hc ((index):72:103958)
    at nu ((index):72:119816)
    at (index):72:121187
```

## User Agent
```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36
```

**Comment by AlexVanderbist** [comment]
Duplicate of #109. Can you please provide some more context in that issue? Thanks

**Comment by FlyingBaum** [comment]
Hey Alex!

Sorry for the late reply, I got off my work shift after opening that ticket.

I looked into issue #109 and made sure to update to the latest version of `spatie/laravel-ignition`.
Then I went to check my frames and whether each one of them contains code snippets:

<img width="470" alt="Screenshot 2022-04-04 at 02 25 38" src="https://user-images.githubusercontent.com/75100411/161455786-77e54a2c-0a1f-45de-a5c1-ee5e389aa9f7.png">

They seem to, what can I do from here on out? Thanks a lot for your help!

Best,
Krist
